teambit.bvm/list/gcp/gcp-list.ts

```diff
@@ -46,13 +55,36 @@ export class GcpList {
     return new RemoteVersionList(remoteVersions);
   }
 
-  async _fetchReleasesList(): Promise<Release[]> {
+  async fetchIndex(): Promise<Release[]> {
     const response = await fetch(BIT_INDEX_JSON, {
       agent: getAgent(BIT_INDEX_JSON, this.proxyConfig),
     });
     return await response.json() as Release[];
   }
 
+  async updateReleaseEntry(version: string, releaseTypeUpdates: Partial<Record<ReleaseType, boolean>>): Promise<Release> {
+    const index = await this.fetchIndex();
+    const release = index.find((release) => release.version === version);
+    if (!release){
+      throw new Error(`version ${version} not found in index.json`);
+    }
+    Object.entries(releaseTypeUpdates).forEach(([releaseType, value]) => {
+      if (value){
+        release[releaseType] = value;
+      } else {
+        delete release[releaseType];
+      }
+    });
+    const newIndex = index.filter((release) => release.version !== version)
+    newIndex.push(release);
```

Comment:
this will change the order in the list. Should we maybe sort the list before updating? Or update the entry at the given index?

Reply:
added sorting